lib/liveValidation/liveValidator.ts

```diff
@@ -196,25 +214,43 @@ export class LiveValidator {
       }
     }
 
-    this.logging("Transforming all specs.");
+    const startGlobalTransform = Date.now();
+    this.logging("Apply global transforms for all specs");
     this.loader.transformLoadedSpecs();
```

Comment:
No need to perf this one. This one is super fast.

Reply:
Removed it.